Log msg changes

OndraZizka opened the pull request:
This makes the log a bit more readable.

mareknovotny commented on reporting/api/src/main/java/org/jboss/windup/reporting/config/Hint.java:
```diff
@@ -284,13 +285,14 @@ private String toString(String title, String text)
+        if (text != null)
+            result.append("\n\t.withText(\"").append(StringUtils.abbreviate(text.trim(), MAX_LOG_TEXT_LENGTH)).append("\")");
+        if (links != null && !links.isEmpty())
+            result.append("\n\t.with(").append(links).append(")");
```
It is better to use `System.lineSeparator()` instead of `\n` to be the same on all OS

jsight replied:
+1 - System.lineSeparator() is better.

OndraZizka replied:
It doesn't really matter anymore, all Windows apps can handle \n. But as you like

mareknovotny commented on reporting/api/src/main/java/org/jboss/windup/reporting/config/Hint.java:
```diff
@@ -187,8 +188,8 @@ public void performParameterized(final GraphRewrite event, final EvaluationConte
-            LOG.info("Hint added to " + locationModel.getFile().getPrettyPathWithinProject() + " [" + this.toString(hintModel.getTitle(), hintText)
-                        + "] with tags: " + StringUtils.join(this.getTags(), " "));
+            LOG.info("Hint added to " + locationModel.getFile().getPrettyPathWithinProject()
+                    + ":\n    " + this.toString(hintModel.getTitle(), null));
```
It is better to use `System.lineSeparator()` instead of `\n` to be the same on all OS.

OndraZizka replied:
Btw how does it matter if our logs are full of newlines coming from our rules?

OndraZizka replied:
Bump. We have \n all over the project, so unless we are determined to fix all of them, I think there is no need to hold this PR because of that.

jsight replied:
@OndraZizka - Fixing the PR is trivial. I agree that we should fix some of the other places where we do this, but that isn't a reason to add more places that need fixing. Just take the 5 minutes to fix and repush. :)

OndraZizka replied:
Ok, but the code is then cluttered with unnecessary calls. The Windows world is now accepting `\n` just ok.

OndraZizka commented:
Rebased and now using the NL constant. Marek's PR not only makes the code harder to read, but also changed the whitespace of certain things. This is addressing the first issue.
